Examples/Run/HepMC3/HepMC3Example.cpp

```diff
@@ -56,37 +56,39 @@ int main(int argc, char** argv) {
   }
   std::cout << std::endl;
 
-  ActsExamples::HepMC3Event simEvent;
-
   std::cout << "Event data:" << std::endl;
   std::cout << "Units: ";
-  if (simEvent.momentumUnit(genevt) == Acts::UnitConstants::GeV)
+  if (ActsExamples::HepMC3Event::momentumUnit(genevt) ==
```

Comment:
I guess you could do `using ActsExamples::HepMC3Event` to make this a bit shorter.

Reply:
Done